### ISSUE TITLE
how to create a workbook from another 

### ISSUE THREAD

**Issue by cwiese** [felipenoris/XLSX.jl#103]
how do I create a workbook from another workbook (i.e. how do I copy a worksheet from one workbook to another? ) 

xf = XLSX.readxlsx("impact_template.xlsx")

newXf = XLSX.openxlsx("new_impact.xlsx", mode="w") do xf
           sheet = xf[1]
           XLSX.rename!(sheet, "new_sheet")
end

 newXf[1] = xf[1]  --> give's an error

**Comment by felipenoris** [comment]
You should use `mode="rw"` (read-write mode). Please refer to https://felipenoris.github.io/XLSX.jl/stable/tutorial/#Edit-Existing-Files-1 .

**Comment by felipenoris** [comment]
Or if you want to use a template and save to another file, you can use `openxlsx` without do-syntax, and write to file using `writexlsx`directly.

Please, refer to the source code for `openxlsx`, reproduced below:

```julia
"""
    openxlsx(filepath; mode="r", enable_cache=true) :: XLSXFile
Supports opening a XLSX file without using do-syntax.
In this case, the user is responsible for closing the `XLSXFile`
using `close` or writing it to file using `XLSX.writexlsx`.
See also [`XLSX.writexlsx`](@ref).
"""
function openxlsx(filepath::AbstractString;
                  mode::AbstractString="r",
                  enable_cache::Bool=true) :: XLSXFile

    _read, _write = parse_file_mode(mode)

    if _read
        @assert isfile(filepath) "File $filepath not found."
        return open_or_read_xlsx(filepath, _write, enable_cache, _write)
    else
        return open_empty_template()
    end
end
```

and the higher level `openxlsx` that uses do-syntax:

```julia
function openxlsx(f::Function, filepath::AbstractString;
                  mode::AbstractString="r", enable_cache::Bool=true)

    _read, _write = parse_file_mode(mode)

    if _read
        @assert isfile(filepath) "File $filepath not found."
        xf = open_or_read_xlsx(filepath, _write, enable_cache, _write)
    else
        xf = open_empty_template()
    end

    try
        f(xf)
    finally

        if _write
            writexlsx(filepath, xf, overwrite=true)
        else
            close(xf)
        end

        # fix libuv issue on windows (#42)
        @static if Sys.iswindows()
            GC.gc()
        end
    end
end
```
```

**Comment by cwiese** [comment]
thank you.
